### PR TITLE
update poll category list, init retroactive categorization mapping

### PR DESCRIPTION
**Pull request by jparklev**

### CODE REVIEW

**Review by ekr696-beep — DISMISSED** [review]

**Comment by jparklev** [comment]
@derek-flossman I updated the list of categories based on your excel sheet. The gov portal has been configured to append the categories from the poll categorization mapping in this PR to each poll according to poll id – this works fine as long as we keep the file up-to-date, but I think the more sustainable solution is to add categories to the poll docs going forward. So I also added category fields to the poll templates. 

cc @LongForWisdom let me know if you have any thoughts on this

Here's what the portal's category filter looks like:
Screen Shot 2020-11-05 at 2 06 05 PM

**Comment by jparklev** [comment]
bump @derek-flossman @LongForWisdom

**Comment by LongForWisdom** [comment]
Didn't realise this was ready to go. Will attempt to get to it this week.

**Comment by jparklev** [comment]
resolved the new merge conflicts with master, should be ready to go again @derek-flossman @LongForWisdom

File: governance/polls/meta/retroactive-poll-categorization.json [hunk 1]
```diff
@@ -0,0 +1,270 @@
+{
```
Comment (derek-flossman) [inline governance/polls/meta/retroactive-poll-categorization.json:1]: "340": ["Collateral"],
"341": ["Collateral"],
"342": ["Collateral"],
"343": ["Collateral"],
"344": ["Collateral"],
"345": ["Collateral"],
"346": ["Collateral"],
"347": ["Collateral"],
"348": ["Collateral"],
"349": ["Collateral"],
"350": ["Collateral"],
"351": ["Oracle"],
"352": ["Oracle"],
"353": ["Technical"],
"354": ["Governance"],
"355": ["Governance"],
"356": ["Collateral"],
"357": ["Governance"],
"358": ["Governance"],
"359": ["Risk Variable"],
"360": ["Risk Variable"],
"361": ["Risk Variable"],
"362": ["Risk Variable"],
"363": ["Risk Variable"],
"364": ["Risk Variable"],
"365": ["Risk Variable"],
"366": ["Technical"],
"367": ["Governance"],
"368": ["Governance"],
"369": ["Governance"],
"370": ["Technical"],
"371": ["Technical"],
"372": ["Governance"],
"373": ["Governance"],
"374": ["Governance"],
"375": ["Collateral"],
"376": ["Collateral"],
"377": ["Risk Variable"],
"378": ["Governance"],
"379": ["Risk Variable"],
"380": ["Risk Variable"],
"381": ["Technical"],
"382": ["Collateral"],
"383": ["Collateral"],
"384": ["Collateral"],
"385": ["Collateral"],
"386": ["Risk Variable"],
"387": ["Risk Variable"],
"388": ["Risk Variable"],
"389": ["Technical"],
"390": ["Collateral"],

Comment (derek-flossman) [reply governance/polls/meta/retroactive-poll-categorization.json:1]: Added these to make the list up-to-date

**Review by derek-flossman — DISMISSED** [review]

File: governance/polls/meta/retroactive-poll-categorization.json (hunk 1 above)
Comment (jparklev) [reply governance/polls/meta/retroactive-poll-categorization.json:1]: added these in 👍  now just need approval from somebody with the right permissions

**Review by LongForWisdom — APPROVED** [review]
Looks good. Thanks @jparklev, sorry it took so long to get this in.

**Comment by jparklev** [comment]
> Looks good. Thanks @jparklev, sorry it took so long to get this in.

no worries, thanks man ~

**Pull request merged**